### ISSUE TITLE
road_following/train_model gets stuck while training

### ISSUE THREAD

**Issue by technodromeltd** [NVIDIA-AI-IOT/jetbot#224]
Im following the road following demo and trying to train my model. Training on host or jetson gets stuck on training,
on line:
for images, labels in iter(train_loader):

There is no error. I have 63 images from previous step for datatset. 
Any ideas whats wrong?
Python 3.6.8
pyTorch 1.0.0

EDIT:

Found this post about DataLoader getting stuck with no errors [discuss.pytorch.org/t/dataloader-iteration-hang-up](https://discuss.pytorch.org/t/dataloader-iteration-hang-up/12886)
Fixed by changing 
`num_workers=0`
